.travis.yml

```diff
@@ -1,20 +1,25 @@
 language: python
 os: linux
 sudo: false
-
 python:
-  - "3.5"
-
+- '3.5'
 before_install:
-  - export PYTHONPATH=$PYTHONPATH:$(pwd)
+- export PYTHONPATH=$PYTHONPATH:$(pwd)
+#- sudo sysctl -w net.ipv6.conf.all.disable_ipv6=1
+#- sudo sysctl -w net.ipv6.conf.default.disable_ipv6=1
 
 install:
-  - pip install -r requirements.txt
-  - npm install newman
-
+- pip install -r requirements.txt
+- npm install newman
 script:
-  - export FLASK_ENV=development
-  - python app.py &
-  - APP_PID=$!
-  - node_modules/.bin/newman run tests/SpartaHack_API_2019.postman_collection.json -e tests/local_env.postman_environment.json
-  - kill $APP_PID
\ No newline at end of file
+- export FLASK_ENV=development
+- export FLASK_APP=app.py
+- flask run &
+- APP_PID=$!
+- node_modules/.bin/newman run tests/SpartaHack_API_2019_tests.postman_collection.json -e tests/SpartaHack-API-2019_dev.postman_environment.json
+- kill $APP_PID
+env:
+  global:
+  - secure: SvXKiUcRo9NadJtgsc4qe6xNtuI2UL+LOveVol/d+fCOUv9Culo3viX2GMSZw3ddZzbDaU3CUyZ96LaJnLtSoixlOE/4+3xmKxqWt56FR04A0HLMn3mi+FqjWhLbKOx9yD5hTikG9ctB6Gr1FQNKrZYTLriSq4/dpAR50yJAKX7o2EfXOdhBKMxTE4CeFs9meErefYvdi94CmFlLLmc7M0yWa5jZe22uAQBsSlP+fb5+SD2+i5QOj2IgdQvc+Rq522/vZm9+XeVnClt39yXOLpH0OueQeR7qpOhjcw6AULSLckAnO7IxFrJhn+mPWvN49Vl/AtsNFSEdK8XEyg9n8kWNQF0Ce0A6FqAX2wY9R4HuwsKok5YluEcI/22zvtjYpQlZFHavmZfneGjohZJ6DTO/Ae23pBj7KFvqEuJ3L4ll0M+2BO8PK8SYLDdrYxgZDZTw4naZWclE1GEwGzYYMUOfkOyK3qcNLp8H4+chHYuUxlZgXKq0yvsoX8hMVg+t/rbbuoMe3Xj4+G1fWwEqIYP4mGxLtgpREKRGCpDtGGb3x6Re1oYOaTuEqL2nYir4OYnlKSnUleQj9G6MOZ0PuyhW5Uwp4P+SvuqEVtXB9ehlA0FQQGaMvbXQq1kf4BhY3uU+359ZF0MOEViWr4q+HuwtBlYrCi64dpZB5J3vz7s=
```

Comment:
I don't think we want these in GH do we?

Reply:
Because that definitely says secure so it won't show in travis, but I think it will show here.